.github/actions/prepare-discovery/action.yml

```diff
@@ -0,0 +1,15 @@
+# A composite action to install discovery
+
+name: Prepare discovery
+
+description: Install discovery
+
+runs:
+  using: composite
+  steps:
+    - name: Install discovery
+      run: |
+        wget https://github.com/mitchnegus/minimega-discovery/releases/download/firewheel-debian_faed761/discovery.deb
+        sudo dpkg -i discovery.deb
+        sudo chown -R $USER:minimega /opt/discovery
```

Comment:
As a note: I did not include dependency installation for either minimega, discovery, or FIREWHEEL in the preparation scripts. I found that each call to `sudo apt update` and `sudo apt install` had enough overhead to increase the test runtime by a decent amount. However, now that the tests are working and I'm not debugging, we may want to consider if that overhead is worth having these composite actions be self-sufficient.